apps/core/src/pages/privacy-policy/index.tsx

```diff
@@ -0,0 +1,370 @@
+  },
+
+  link: {
+    boxSizing: "border-box",
+    display: "flex",
+    alignItems: "center",
+    textDecoration: "none",
+    borderTopRightRadius: theme.radius.md,
+    borderBottomRightRadius: theme.radius.md,
+    color:
+      theme.colorScheme === "dark"
+        ? theme.colors.dark[0]
+        : theme.colors.gray[7],
+    padding: `0 ${theme.spacing.md}px`,
+    fontSize: theme.fontSizes.sm,
+    marginRight: theme.spacing.md,
+    fontWeight: 500,
+    height: 44,
+
+    "&:hover": {
+      backgroundColor:
+        theme.colorScheme === "dark"
+          ? theme.colors.dark[5]
+          : theme.colors.gray[1],
+      color: theme.colorScheme === "dark" ? theme.white : theme.black
+    }
+  },
+
+  linkActive: {
+    "&, &:hover": {
+      borderLeftColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      backgroundColor: theme.fn.variant({
+        variant: "filled",
+        color: theme.primaryColor
+      }).background,
+      color: theme.white
+    }
+  }
+}));
+
+// breadcrumbs
+const items = [
+  { title: "Privacy & Terms ", href: "#" },
+  { title: "Privacy policy", href: "#" }
+].map((item, index) => (
+  <Anchor href={item.href} key={index}>
+    {item.title}
+  </Anchor>
+));
+
+const linksMockdata = [
+  { title: "The Scope of This Privacy Policy", id: "privacyScope" },
+  { title: "Information we collect", id: "informationCollection" },
+  {
+    title: "How We Process Your Information, and the Laws That Support It",
+    id: "informationProcessing"
+  },
+  { title: "Information we share", id: "information-sharing" },
+  { title: "Data Retention", id: "data-retention" },
+  { title: "Security", id: "security" },
+  {
+    title: "Modifications to this privacy policy",
+    id: "privacyModification"
+  },
+  { title: "Our global operations", id: "global-ops" },
+  { title: "Your Rights", id: "rights" },
+  { title: "Data Protection Authority", id: "data-auth" },
+  { title: "Limitation of liability", id: "limitation" },
+  {
+    title: "Member account, Password, and Security",
+    id: "member"
+  },
+  { title: "Use of services", id: "service-use" },
+  { title: "Privacy Policy", id: "privacy" }
+];
+
+const PrivacyPage = () => {
+  const dispatch = useAppDispatch();
+
+  dispatch(uiSliceActions.setSidebarVisibility(false));
+  const { classes, cx } = useStyles();
+  const [active, setActive] = useState("Content");
+  const [activeLink, setActiveLink] = useState("privacyScope");
+
+  const links = linksMockdata.map(link => (
+    <a
+      className={cx(classes.link, {
+        [classes.linkActive]: activeLink === link.id
+      })}
+      href={`#${link.id}`}
+      onClick={() => {
+        // event.preventDefault();
+        setActiveLink(link.id);
+      }}
+      key={link.id}
+    >
+      {link.title}
+    </a>
+  ));
+
+  const headerRef = useRef<HTMLDivElement>(null);
+  const sectionWrapperRef = useRef<HTMLDivElement>(null);
+  const refs = {
```

Comment:
i think this ref passing might be a messy

do you think we can solve this in another way?